examples/data_sources/leanspace_leaf_space_ground_stations_links.tf

```
@@ -1,4 +1,4 @@
-data "leanspace_leaf_space_ground_station_links" "all" {
+data "leanspace_leaf_space_ground_stations_links" "all" {
```

I would expect only one plural i.e.:
```suggestion
data "leanspace_leaf_space_ground_station_links" "all" {
```

---

Ok done here and for satellite too